python/tk_hiero_export/version_creator.py

```diff
@@ -128,6 +129,27 @@ def __init__(self, initDict):
         self._quicktime_path = None
         self._temp_quicktime = None
 
+    def addWriteNodeToScript(self, script, rootNode, framerate):
+        """
+        Override the default addWriteNodeToScript functionality so that we can add a SetNode before
+        the default writenodes get added to the script. The SetNode will allow us to later push our mov writenode
+        to this point so that it get parented correctly.
+        :param script:
+        :param rootNode:
+        :param framerate:
+        :return:
+        """
+        self.app.log_debug("Adding SetNode before base write node gets added")
+        # We create and store an arbitrary name for our SetNode, this is used as an id, so that when we create
+        # the PushNode later we can connect it to this SetNode.
+        self.write_set_node_label = "SG_Write_Attachment"
+
+        # Add the SetNode before the write nodes are added.
+        setCommand = nuke.SetNode(self.write_set_node_label, 0)
```

Comment:
[nitpick] I know we're stuck between Nuke's mixed-case convention and our pep8-compliant underscores, but let's stick with `set_command` instead of `setCommand`.